tmpl/install/redhat/centcore.systemd

```diff
@@ -18,11 +18,17 @@
 
 [Unit]
 Description=Centreon Core
+PartOf=centreon.service
+After=centreon.service
+After=network-online.target
```

Comment:
IMHO the _network-online_ and _network_ targets should not be added to the unit.